web/base_viewer.js

```diff
@@ -554,7 +554,7 @@ class BaseViewer {
       if (evt.cssTransform || this._onePageRenderedCapability.settled) {
         return;
       }
-      this._onePageRenderedCapability.resolve();
+      this._onePageRenderedCapability.resolve({ timestamp: evt.timestamp });
```

Comment:
One thing that I'm wondering is if we could potentially just do the following instead:
```javascript
      this._onePageRenderedCapability.resolve({ timestamp: performance.now() });
```
and then remove the `timestamp: performance.now(),`-lines from the various "pagerendered"-events in the https://github.com/mozilla/pdf.js/blob/master/web/pdf_page_view.js file.

It seems totally unnecessary to do these calls *potentially twice* for each rendered page, when we're not using anything except the very first one anyway.